meta/main.yml

```diff
@@ -0,0 +1,22 @@
+galaxy_info:
+  author: Insert author(s) here
```

Comment:
That's not correct, also correct the rest of the file

Reply:
Must've forgot to fill it out.